src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -150,6 +150,36 @@ include::partial$handle-constraints.adoc[]
 
 |===
 
+[#generate-syhtetic-data]
```

Comment:
You don't need to provide an ID if you're not going to use it.
You can keep it but it doesn't change much